cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -497,7 +496,7 @@ parameter_types! {
 parameter_types! {
 	pub const CreateAssetCall: [u8;2] = [53, 0];
 	pub const CreateAssetDeposit: u128 = (UNITS / 10) + EXISTENTIAL_DEPOSIT;
-	pub const InboundQueuePalletInstance: u8 = parachains_common::rococo::snowbridge::INBOUND_QUEUE_PALLET_INDEX;
+	pub const InboundQueuePalletInstance: u8 = testnet_parachains_constants::rococo::snowbridge::INBOUND_QUEUE_PALLET_INDEX;
```

Comment:
cleaner if imported on top with `use testnet_parachains_constants::rococo`

Reply:
ok, thank you, fixed here: https://github.com/paritytech/polkadot-sdk/pull/3134/commits/406c5bf199a90838bb90dcd23d7bea89eb434ce9